### DISCUSSION TITLE
Release v1.0.8 / 5.55.8

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1554] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixed many issues, and adds a new functionality: "BreakoutProcess=program.exe" which allow to preset programs to be able to escape a sandbox, hence this is a feature rather for compartmentalization then security, but the way its implemented, a breakout process, will be captured by an other sandbox if it is configured as a forced process for it. So a possibly security related use case would be to have the a box dedicated to run your web browser only, where it is forced, and have it configured as a break out process for all other boxes or globally. In this scenario no mater what boxed or unboxed application starts a browser it will always run in the browser box. 

This new feature is enabled only for certified project supporters, if I reach 250 patrons it will be made available to all users, please consider supporting the development of sandbox-plus: https://www.patreon.com/DavidXanatos

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- added Portuguese of Portugal on Plus UI (by JNylson, isaak654, mpheath) [#1497](https://github.com/sandboxie-plus/Sandboxie/pull/1497)
- added "BreakoutProcess=program.exe", with this option selected applications can be started unboxed from within a box [#1500](https://github.com/sandboxie-plus/Sandboxie/issues/1500)
-- the program image must be located outside the sandbox for this to work
-- if another sandbox has "ForceProcess=program.exe" configured, it will capture the process
-- use case: set up a box with a Web browser forced, when another box opens a website, this will happen in the dedicated browser box
-- Note: "BreakoutFolder=some\path" is also available
- added silent uninstall switch `/remove /S` for Classic installer (by sredna) [#1532](https://github.com/sandboxie-plus/Sandboxie/pull/1532)

### Changed
- The filename "sandman_pt" was changed to "sandman_pt_BR" (Brazilian Portuguese) [#1497](https://github.com/sandboxie-plus/Sandboxie/pull/1497)
- The filename "sandman_ua" was changed to "sandman_uk" (Ukrainian) [#1527](https://github.com/sandboxie-plus/Sandboxie/issues/1527)
-- Note: Translators are encouraged to follow the [Localization notes and tips](https://github.com/sandboxie-plus/Sandboxie/discussions/1123#discussioncomment-1203489) before creating a new pull request
- updated Firefox update blocker (discovered by isaak654) [#1545](https://github.com/sandboxie-plus/Sandboxie/issues/1545#issuecomment-1013807831)

### Fixed
- fixed issue with opening all file access OpenFilePath=* [#971](https://github.com/sandboxie-plus/Sandboxie/issues/971)
- fixed issue with opening network shares [#1529](https://github.com/sandboxie-plus/Sandboxie/issues/1529)
- fixed possible upgrade issue with Classic installer (by isaak654) [130c43a](https://github.com/sandboxie-plus/Sandboxie/commit/130c43a62c9778b734fa625bf4f46b12d0701719)
- fixed minor issues with Classic installer (by sredna) [#1533](https://github.com/sandboxie-plus/Sandboxie/pull/1533)
- fixed issue with Ldr_FixImagePath_2 [#1507](https://github.com/sandboxie-plus/Sandboxie/issues/1507)
- when using "Run Sandboxed" with SandMan UI and the UI is off, it wil stay off.
- fixed issue with Util_GetProcessPidByName that should resolve the driver sometimes failing to start at boot [#1451](https://github.com/sandboxie-plus/Sandboxie/issues/1451)
- SandMan will now run in background like SbieCtrl when starting a boxed process [post506](https://forum.xanasoft.com/viewtopic.php?p=506#p506)
- fixed taskbar not showing with persistent box border in full screen [post474](https://forum.xanasoft.com/viewtopic.php?p=474#p474)
- fixed box border not spanning across multiple monitors [#1512](https://github.com/sandboxie-plus/Sandboxie/issues/1512)
- fixed issues with border when using DPI scaling [#1506](https://github.com/sandboxie-plus/Sandboxie/issues/1506)
- fixed DPI issues with Qt [#1368](https://github.com/sandboxie-plus/Sandboxie/issues/1368)
- fixed issue with bright flashing on window creation when in dark mode [#1231](https://github.com/sandboxie-plus/Sandboxie/issues/1231)
- fixed issues with the PortableRootDir setting [#1509](https://github.com/sandboxie-plus/Sandboxie/issues/1509)
- fixed issue with the settings window crashing when the driver was not connected
- fixed DPI issues with Finder Tool [#912](https://github.com/sandboxie-plus/Sandboxie/issues/912)
- fixed another issue with reused process IDs [#1547](https://github.com/sandboxie-plus/Sandboxie/issues/1547)
- fixed issue introduced in 1.0.6 related to SeAccessCheckByType [#1548](https://github.com/sandboxie-plus/Sandboxie/issues/1548)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.8'>Release v1.0.8 / 5.55.8</a>.</em>

**Comment by BilleBarrett** [comment]
I have been receiving calls about the download recovery windows on the Plus version not opening up all the time, especially when PDF files are downloaded 
Is there a setting that will make it open, there is some confusion when or where data is going 
Thanks
Bill e. Barrett

**Reply by ximGBu4cyQss5P** [reply]
I've experienced this on multiple machines for the last couple of months across multiple versions.  Immediate Recovery is enabled, but sometimes the dialog just never appears, and you have to manually open the Sandboxie-Plus UI and recover them.  I've been meaning to dig into it but haven't had time.

**Comment by BilleBarrett** [comment]
This is most prevalent in the Plus version and it is freeking out some of my client that I’m trying to get off of the Classic version

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
This may be trivial, but could the version number be added to the Sandboxie Classic header, it would make it easier to determine what version my client is using.
Thanks
Bill

**Comment by DavidXanatos** [comment]
header? you mean the title bar? sure will do that if I don't forget


there is a setting for teh recovery window:
![grafik](https://user-images.githubusercontent.com/3890945/150550550-ddfaecce-6962-4766-9301-3eee7cc06701.png)

and an other:

![grafik](https://user-images.githubusercontent.com/3890945/150550639-dfce808a-a0e8-4602-9922-3254305747d0.png)

please check if booth are enabled to have the behavior of the classic sandboxie

**Comment by BilleBarrett** [comment]
This was not checked, it is now

 <https://user-images.githubusercontent.com/3890945/150550639-dfce808a-a0e8-4602-9922-3254305747d0.png> 

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
Thank you

That will make it easier on me to check where clients are

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
I am trying to force Adobe Reader [64] to open in the Sandbox, and I can’t seem to do it
I tried to use the ProcessGroup and ForceProcess but that didn’t seem to work
I find I must right click on the PDF and tell it to run in the Sandbox
What am I doing wrong?

Thanks Bill



ProcessGroup=<PDFs>,"C:\Program Files\Adobe\Acrobat DC\Acrobat\Acrobat.exe"

ProcessGroup=<StartRunAccess>,

ForceProcess="C:\Program Files\Adobe\Acrobat DC\Acrobat\Acrobat.exe"

 

I also have the Templates on

Template=AdobeAcrobatReader

Template=Adobe Acrobat

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by isaak654** [comment]
It works fine with ForceProcess=Acrobat.exe

**Comment by BilleBarrett** [comment]
Not trying to get personnel, but I must be doing something wrong
With old Sandboxie I have never had this problem, forcing programs in to the sandbox

I just installed Sandboxie-Plus on a new computer window 11 that I just finished building

These are my setup settings, after I uninstalled Sandboxie-Plus, deleted Sadnboxie.ini
removed all Sandbox folder and did a fresh install

I did this setting 1st [next window]



Then I setup this window [Nest window]



This part of the Sandboxie.ini file



I then click on a pdf program and this is what I got “NO sandbox”



The I right clicked on the same pdf and told it to run in the Sandbox 

 



And it did exactly the way it should if I just clicked on the pdf file, it has always done it in the past



Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
Ok so I uninstalled Sandboxie-Plus and cleaned up any residuals

Rebooted the computer and then installed Sandboxie-classic and made these settings below 

Now I open a PFD file in the Sandbox and as you can see it open OK
I will now install Sandboxie-Plus and see if it will work
After installing Sandboxie-Plus, rebooting the system, and opening a PDF file it WORKS
Go figure 

 




When I open a pdf files this appears


Closed the pdf 


This is Plus-installed 





Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by walmar** [comment]
I am not satisfied with the following last change.
> „SandMan will now run in background like SbieCtrl when starting a boxed process“

As I already [commented to Release v0.8.8 / 5.50.8](https://github.com/sandboxie-plus/Sandboxie/discussions/998#discussioncomment-1133634):
> Version 0.8.2 of Sandboxie-Plus was running minimized to tray via Run Sandboxed context menu option with Sandboxie.ini UserSettings section option SbieCtrl_AutoStartAgent=SandMan.exe -autorun.
> Sandboxie-Plus v0.8.8 Run Sandboxed context menu option seems to ignore the UserSettings section option SbieCtrl_AutoStartAgent.

After the change, "Run Sandboxed" behaves differently as SbieCtrl.exe when used several times.
- System start
  1. RS option > close SandMan or SbieCtrl
  2. RS option (again)
     - with <1.0.8/5.55.8:  SandMan/SbieCtrl is running again
     - with 1.0.8: SandMan.exe is missing

**Reply by isaak654** [reply]
Reposted in https://github.com/sandboxie-plus/Sandboxie/issues/1579

**Comment by BilleBarrett** [comment]
I do have these changes; it is weird that sometime it will happen and mostly on pdf files I must go get them
BTW is there way I can set the Hotkey for terminating all boxed Processes in the ini file
I have a short cut for Classic which is Ctrl+ALT+T that call to Sandboxie to terminate all 

and I can’t set that when I do a remote install, just asking 😊

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by DavidXanatos** [comment]
The terminate all boxed processes is a feature of the sandman UI and needs to be configured in the Sandboxie-plus.ini
```
[Options]
EnablePanicKey=true
PanicKeySequence=Ctrl+Cancel
```

**Comment by BilleBarrett** [comment]
I just did and it didn’t change
I remove the sequence from the global setting and then edited Sandboxie.ini in windows

At the end of the ini file, but it didn’t install it ? 
Sorry
#

# Sandboxie-Plus configuration file

#

[GlobalSettings]

Template=WindowsRasMan

Template=WindowsLive

Template=KeyScrambler

Template=OfficeLicensing

Template=OfficeClickToRun

Template=RoboForm

FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\%SANDBOX%

SeparateUserFolders=y

KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%

IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%

NetworkEnableWFP=n

EnableObjectFiltering=n

EnableWin32kHooks=y

EditAdminOnly=n

ForceDisableAdminOnly=n

ForgetPassword=n

 

[DefaultBox]

ConfigLevel=9

BlockNetworkFiles=y

Template=nVidia_Stereoscopic3D

Template=nVidia_nView

Template=WindowsDefender

Template=Windows_Live_Mail

Template=Office_Outlook

Template=Chrome_Profile_DirectAccess

Template=Chrome_Sync_DirectAccess

Template=Chrome_Preferences_DirectAccess

Template=Chrome_Passwords_DirectAccess

Template=Chrome_Cookies_DirectAccess

Template=Chrome_History_DirectAccess

Template=Chrome_Bookmarks_DirectAccess

Template=Chrome_Force

Template=Firefox_Profile_DirectAccess

Template=Firefox_Session_DirectAccess

Template=Firefox_Passwords_DirectAccess

Template=Firefox_Cookies_DirectAccess

Template=Firefox_Bookmarks_DirectAccess

Template=Firefox_Force

Template=Edge_Profile_DirectAccess

Template=Edge_Phishing_DirectAccess

Template=Edge_Sync_DirectAccess

Template=Edge_Preferences_DirectAccess

Template=Edge_Passwords_DirectAccess

Template=Edge_Cookies_DirectAccess

Template=Edge_History_DirectAccess

Template=Edge_Bookmarks_DirectAccess

Template=Edge_Force

Template=OpenSmartCard

Template=OpenBluetooth

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=AutoRecoverIgnore

RecoverFolder=M:\Downloads

RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%

RecoverFolder=%Personal%

RecoverFolder=%Desktop%

BorderColor=#0000FF,ttl,6

Enabled=y,Administrator,BilleB

BoxNameTitle=y

AutoDelete=y

NeverDelete=n

ProcessGroup=<Adobe Readers>,acrord32.exe,acrobat.exe

LingerProcess=acrobat.exe

CopyLimitKb=249152

ForceProcess=acrobat.exe

ForceProcess=acrord32.exe

ForceFolder=W:\

AutoRecover=y

 

[UserSettings_10FE026B]

SbieCtrl_UserName=billeb

SbieCtrl_BoxExpandedView=DefaultBox

SbieCtrl_NextUpdateCheck=1643646531

SbieCtrl_AutoApplySettings=y

SbieCtrl_SettingChangeNotify=n

SbieCtrl_HideWindowNotify=n

SbieCtrl_WindowCoords=-1680,250,1237,632

SbieCtrl_ActiveView=40021

SbieCtrl_TerminateWarn=n

SbieCtrl_AutoStartAgent=SandMan.exe

SbieCtrl_EnableAutoStart=y

 

[Options]

EnablePanicKey=true

PanicKeySequence=CTRL+ALT+T

AutoRecover=n

BlockNetworkFiles=y

RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%

RecoverFolder=%Personal%

RecoverFolder=%Desktop%

BorderColor=#00FFFF,ttl

Template=OpenBluetooth

Template=SkipHook

Template=FileCopy

Template=qWave

Template=BlockPorts

Template=LingerPrograms

Template=Chrome_Phishing_DirectAccess

Template=Firefox_Phishing_DirectAccess

Template=AutoRecoverIgnore

ConfigLevel=9

 

[Options]

EnablePanicKey=true

PanicKeySequence=Ctrl+ALT+T

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by isaak654** [comment]
It looks like you pasted the suggested lines in the wrong ini file.
Sandboxie Plus has the following ini files in different paths:
* `C:\Users\Username\AppData\Local\Sandboxie-Plus\Sandboxie-Plus.ini` 
* `C:\Windows\Sandboxie.ini`

**Comment by BilleBarrett** [comment]
Is there a different path for each user if multiple users

 

C:\Users\%Username%\AppData\Local\Sandboxie-Plus\Sandboxie-Plus.ini

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by DavidXanatos** [comment]
Yes each user has an own ini path,
shell I add an option for corporate users to have the Sandboxie-Plus.ini ini shared among all users in a central location like C:\ProgramData\Sandboxie-Plus  ?

**Comment by BilleBarrett** [comment]
That would be nice, I could control users Sandboxie without having to log in as the user and edit Sandboxie 
Let me think on that 
Thank you

Bill

PS remember to put the revision lever on Classic header 😊

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Reply by DavidXanatos** [reply]
Ok will add that to the next build, installation with a for business cert will load the Sandboxie-Plus.ini from C:\ProgramData\Sandboxie-Plus\  instead of C:\Users\%Username%\AppData\Local\Sandboxie-Plus\ 

Anything else i could add to improve the administrability?

**Comment by BilleBarrett** [comment]
All I have to do is remember it, after all I’m almost 75 😊 btw my 55th year as a computer tech

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891   ***@***.***
"Your operating system and Data are only as good as your last backup"
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
